core/templates/dev/head/domain/statistics/PlaythroughObjectFactorySpec.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the PlaythroughObjectFactory.
+ */
+
+describe('Playthrough Object Factory', function() {
+  beforeEach(module('oppia'));
+
+  beforeEach(inject(function($injector) {
+    this.PlaythroughObjectFactory = $injector.get(
+      'PlaythroughObjectFactory');
+    this.LearnerActionObjectFactory = $injector.get(
```

Comment:
nit: consider making this an acronym since it isn't really the focus of these tests:

```javascript
this.LAOF = $injector.get('LearnerActionObjectFactory');

expect(action).toEqual(this.LAOF.createFromBackendDict('...'));
```

I don't feel too strongly about this, both sides of the coin have their advantages. Pick whichever you think is best!

Reply:
I do like this better. Done and extended to all specs.